aries-site/src/examples/templates/forms/SingleFormFieldExample.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Text,
+  TextInput,
+} from 'grommet';
+
+export const SingleFormFieldExample = () => {
+  const [cancel, setCancel] = React.useState('');
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  const confirmDelete = () => {
+    const doesMatch = cancel.delete === 'DELETE';
+    return doesMatch
+      ? undefined
+      : { message: 'Verification Incorrect', status: 'error' };
```

Comment:
Wondering if we can have a more human sounding verification message? Ideas:
- Text needs to match "DELETE".
- Text does not match "DELETE".
- etc.?

Reply:
Thats fair Ill update